ext/simplexml/tests/simplexml_import_dom_basic.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+simplexml_import_dom() - Basic test on invalid nodetype error
+--SKIPIF--
```

Comment:
You should use the EXTENSIONS section: https://github.com/php/php-src/pull/6787

Reply:
I like this change. I'll be delighted to.